lib/Yam/agama/agama_base.pm

```diff
@@ -9,19 +9,13 @@ use base 'opensusebasetest';
 use strict;
 use warnings;
 use testapi;
-use y2_base 'save_upload_y2logs';
 use Utils::Logging 'save_and_upload_log';
-use utils 'ensure_serialdev_permissions';
-use serial_terminal qw(select_serial_terminal);
-
-sub pre_run_hook {
-    $testapi::password = 'linux';
-}
 
 sub post_fail_hook {
     select_console 'root-console';
     # "agama logs store" gathers output from dmesg, journalctl and y2logs.
     save_and_upload_log('agama logs store', "/tmp/agama-logs.tar.gz");
+    save_and_upload_log('journalctl -b > /tmp/journal.log', "/tmp/journal.log");
     upload_traces();
 }
 
```

Comment:
we don't need this post_run_hook anymore.

Reply:
Then how to deal with the logs when failed, add post_fail_hook for each test modules?

Reply:
post_run_hook doesn't run on failure, I'm referring to 
https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/20078/files#diff-c0284caaef4805452f31bdbbbbf8ca05e624c9e999e5cd782308594e54299475R22-R26

Reply:
Ok, I already removed that. I think you add comments in wrong line.

Reply:
Updated and removed the post_run_hook.